### ISSUE TITLE
[stable/airflow] Worker CrashLoopBackOff running chart version 2.8.7

### ISSUE THREAD

**Issue by luizferreira094** [helm/charts#14559]
**Describe the bug**
I was using this chart with version 2.8.6, after i upgrade to 2.8.7, the worker pods can't run, it restarts all the time.
Here's the **worker** log:
```
waiting 60s...
executing worker...
[2019-06-06 14:38:41,905] {settings.py:174} INFO - settings.configure_orm(): Using pool settings. pool_size=5, pool_recycle=1800, pid=1
[2019-06-06 14:38:42,444] {__init__.py:51} INFO - Using executor CeleryExecutor

```

**Version of Helm and Kubernetes**:
**Helm**: 
```
Client: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
Server: &version.Version{SemVer:"v2.13.1", GitCommit:"618447cbf203d147601b4b9bd7f8c37a5d39fbb4", GitTreeState:"clean"}
```

**Kubernetes**:
```
Client Version: version.Info{Major:"1", Minor:"14", GitVersion:"v1.14.1", GitCommit:"b7394102d6ef778017f2ca4046abbaa23b88c290", GitTreeState:"clean", BuildDate:"2019-04-08T17:11:31Z", GoVersion:"go1.12.1", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"13+", GitVersion:"v1.13.6-gke.5", GitCommit:"46b3ddd492a78deae773f24b47c5c564a9b928b1", GitTreeState:"clean", BuildDate:"2019-05-18T01:19:43Z", GoVersion:"go1.11.5b4", Compiler:"gc", Platform:"linux/amd64"}
```

**Which chart**: 2.8.7

**What happened**:
Worker pods restarting

**What you expected to happen**:
Worker pods running

**How to reproduce it** (as minimally and precisely as possible):
```
helm upgrade --install --version 2.8.7 airflow stable/airflow --namespace airflow
```

**Anything else we need to know**:

**Comment by luizferreira094** [comment]
Apparently  a dependency was conflicting with airflow, closing it.

**Comment by rmmcnulty9** [comment]
How can this be fixed? I am on the chart version 3.0.0, and with the docker image 1.10.3 I still get this silent crash. If I am on 3.0.0 with 1.10.2 I get this error: https://github.com/puckel/docker-airflow/issues/342

What is the combination of versions that is stable and working currently for Airflow (helm chart)?
